astropath/bayesian.py

```diff
@@ -2,48 +2,56 @@
 
 import numpy as np
 
-import copy
-
 from astropy import units
 from astropy.coordinates import SkyCoord
 
+from astropath import chance
+
 from IPython import embed
 
 
-def raw_prior_Oi(Pchance, Sigma_m, method, half_light=None):
+def raw_prior_Oi(method, mag, half_light=None, Pchance=None):
     """
     Raw prior for a given set of Pchance values and/or n(m)
```

Comment:
Would be useful to mention that it uses Driver et al 2016 to estimate Sigma_m from the magnitude of the galaxy.

Reply:
done